src/cockpit/bridge.py

```diff
@@ -107,8 +108,32 @@ def __init__(self, args: argparse.Namespace):
             self.peers_rule,
         ])
 
+    def info(self) -> JsonObject:
+        pw = pwd.getpwuid(os.getuid())
```

Comment:
code duplication of the user stuff, but looking the other way as (1) it's not an identical copy and (2) we can drop the D-Bus implementation at some point